lib/ledgerizer/util/validators.rb

```diff
@@ -3,10 +3,16 @@ module Validators
     include Ledgerizer::Formatters
     include Ledgerizer::Errors
 
-    def validate_active_record_model_name!(model_name, error_prefix)
-      return true if ActiveRecord::Base.model_names.include?(model_name)
+    def validate_ledgerized_class_name!(value, error_prefix, ledgerizer_mixin)
+      klass = format_string_to_class(value)
 
-      raise_error("#{error_prefix} must be an ActiveRecord model name")
+      if klass.blank?
+        raise_error("#{error_prefix} must be a snake_case representation of a Ruby class")
+      end
+
+      return true if klass.ancestors.include?(ledgerizer_mixin)
+
+      raise_error("#{error_prefix} must include #{ledgerizer_mixin}")
```

Comment:
Estaba pensando que para este tipo de validaciones quedarían mucho mas claras usando algo tipo 
`assert` pero aparentemente eso no existe en ruby jaja (es lo que se hace tipicamente en las librerías de python para validar los inputs del usuario de la librería).  Es como lo que hacen en esta gema. https://github.com/jorgemanrubia/solid_assert

Reply:
acá hay mas info https://stackoverflow.com/questions/147969/is-it-idiomatic-ruby-to-add-an-assert-method-to-rubys-kernel-class

Reply:
mmm para hacerlo así tendría que implementar a mano el módulo Assertions como sugieren por ahí. Prefiero dejarlo así por ahora.